docs/docs/external-volumes.md

```diff
@@ -0,0 +1,137 @@
+---
+title: Stateful Applications Using External Persistent Volumes
+---
+
+# Stateful Applications Using External Persistent Volumes
+
+<div class="alert alert-danger" role="alert">
+  <span class="glyphicon glyphicon-exclamation-sign" aria-hidden="true"></span> Adapted in Marathon Version 1.0 <br/>
+  The Persistent Storage functionality is considered beta, so use this feature at your own risk. We might add, change, or delete any functionality described in this document.
```

Comment:
The "External Persistent Storage functionality"? Or is External Temporary Storage stable/GA?